editor.planx.uk/src/pages/FlowEditor/components/Team/tests/mocks/mockUsers.tsx

```diff
@@ -7,3 +9,47 @@ export const alreadyExistingUser: TeamMember = {
   id: 3,
   role: "teamEditor",
 };
+
+export const emptyTeamMemberObj: TeamMember = {
+  firstName: "",
+  lastName: "",
+  email: "",
+  id: 3,
+  role: "teamEditor",
+};
+
+export const mockPlatformAdminUser: User = {
+  isPlatformAdmin: true,
+  firstName: "r",
+  lastName: "r",
+  email: "r",
+  id: 909,
+  teams: [
+    {
+      role: "teamEditor",
+      team: {
+        name: "planX",
+        slug: "planx",
+        id: 0,
+      },
+    },
+  ],
+};
+
+export const mockPlainUser: User = {
```

Comment:
Nice :) you could reduce duplication even further by spreading `mockPlainUser` into `mockPlatformAdminUser` and just changing the `isPlatformAdmin` boolean, e.g.

```
export const mockPlatformAdminUser: User = {...mockPlainUser, isPlatformAdmin: true }
```